src/layouts/default.layout.jsx

```diff
@@ -14,7 +14,7 @@ class DefaultLayout extends PureComponent {
                             <Redirect to='/auth/signin' />
                         </Route>
                         <Route path='/auth/signup' component={Signup} />
-                        <Route path='/auth/signin' component={Signin} />
```

Comment:
@mujohn26 just like what @Jajabenit250 said, you cannot edit this file, your component should also be in `Signin` component